packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx

```diff
@@ -314,13 +360,115 @@ export default function LabeledField(props: Props) {
+            return null;
+        }
+        return (
+            <BodyText
+                style={[
+                    styles.description,
+                    styles.helperTextMessage,
+                    styles.textWordBreak,
+                    stylesProp?.elementBeforeFieldStart,
+                ]}
+                tag="div"
+                id={elementBeforeFieldStartId}
+                testId={testId && `${testId}-element-before-field-start`}
+            >
+                {elementBeforeFieldStart}
+            </BodyText>
+        );
+    }
+
+    function maybeRenderElementBeforeFieldEnd() {
+        if (!elementBeforeFieldEnd) {
+            return null;
+        }
+        return (
+            <BodyText
+                style={[
+                    styles.description,
+                    styles.helperTextMessage,
+                    styles.textWordBreak,
+                    stylesProp?.elementBeforeFieldEnd,
+                ]}
+                tag="div"
+                id={elementBeforeFieldEndId}
+                testId={testId && `${testId}-element-before-field-end`}
+            >
+                {elementBeforeFieldEnd}
+            </BodyText>
+        );
+    }
+
+    function maybeRenderElementAfterFieldStart() {
+        if (!elementAfterFieldStart) {
+            return null;
+        }
+        return (
+            <BodyText
+                style={[
+                    styles.helperTextSectionWithContent,
+                    styles.helperTextMessage,
+                    styles.textWordBreak,
+                    stylesProp?.elementAfterFieldStart,
+                ]}
+                tag="div"
+                id={elementAfterFieldStartId}
+                testId={testId && `${testId}-element-after-field-start`}
+            >
+                {elementAfterFieldStart}
+            </BodyText>
+        );
+    }
+
+    function maybeRenderElementAfterFieldEnd() {
+        if (!elementAfterFieldEnd) {
+            return null;
+        }
+        return (
+            <BodyText
+                style={[
+                    styles.helperTextSectionWithContent,
+                    styles.helperTextMessage,
+                    styles.textWordBreak,
+                    hasError && styles.error,
+                    stylesProp?.elementAfterFieldEnd,
+                ]}
+                tag="div"
+                id={elementAfterFieldEndId}
+                testId={testId && `${testId}-element-after-field-end`}
+            >
+                {elementAfterFieldEnd}
+            </BodyText>
+        );
+    }
+
     return (
         <View style={stylesProp?.root}>
             {renderLabel()}
-            {maybeRenderDescription()}
+            <View
+                style={[
+                    styles.helperTextContainer,
+                    styles.beforeHelperTextContainer,
+                ]}
+            >
+                <View>
+                    {maybeRenderDescription()}
```

Comment:
In the next PR, I'm going to remove the `description` prop. The `elementBeforeFieldStart` can be used instead since they pretty much do the same thing (this will be a breaking change!)